<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Valgrind::CachegrindConfigPage</class>
 <widget class="QWidget" name="Valgrind::CachegrindConfigPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>494</width>
    <height>369</height>
   </rect>
  </property>
  <layout class="QVBoxLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string/>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QCheckBox" name="kcfg_cacheSim">
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Enables or disables collection of cache access and miss counts.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="text">
         <string>Cache simulation</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="kcfg_branchSim">
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Enables or disables collection of branch instruction and misprediction counts. By default this is disabled as it slows Cachegrind down by approximately 25%. Note that you cannot specify &lt;span style=&quot; font-family:'Monospace'; font-weight:600; font-style:italic;&quot;&gt;--cache-sim=no&lt;/span&gt; and &lt;span style=&quot; font-family:'Monospace'; font-weight:600; font-style:italic;&quot;&gt;--branch-sim=no&lt;/span&gt; together, as that would leave Cachegrind with no information to collect.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="text">
         <string>Branch simulation</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="KMessageWidget" name="messageWidget">
        <property name="text">
         <string>You cannot disable both cache and branch simulation, as that would leave Cachegrind with no information to collect.</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
        <property name="closeButtonVisible">
         <bool>false</bool>
        </property>
        <property name="messageType">
         <enum>KMessageWidget::Warning</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QFormLayout" name="argsLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="extraArgsLabel">
          <property name="text">
           <string>Extra parameters:</string>
          </property>
          <property name="buddy">
           <cstring>kcfg_extraArgs</cstring>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="kcfg_extraArgs">
          <property name="clearButtonEnabled">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="cgAnnotateArgsLabel">
          <property name="text">
           <string>cg_annotate parameters:</string>
          </property>
          <property name="buddy">
           <cstring>kcfg_cgAnnotateArgs</cstring>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="kcfg_cgAnnotateArgs">
          <property name="clearButtonEnabled">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KMessageWidget</class>
   <extends>QFrame</extends>
   <header>kmessagewidget.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>kcfg_cacheSim</tabstop>
  <tabstop>kcfg_branchSim</tabstop>
  <tabstop>kcfg_extraArgs</tabstop>
  <tabstop>kcfg_cgAnnotateArgs</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
